Pear/Supporting/Keys.swift

```diff
@@ -16,6 +16,7 @@ struct Keys {
     static let announcementsScheme = Keys.keyDict["announcements-scheme"] as! String
     static let googleClientID = Keys.googleServiceDict["CLIENT_ID"] as? String ?? ""
     static let serverURL = Keys.keyDict["SERVER_URL"] as? String ?? ""
+    static let appdevServerURL = Keys.keyDict["APPDEV_SERVER_URL"] as? String ?? ""
```

Comment:
very nitpicky but maybe rename ```APPDEV_SERVER_URL``` since it sounds similar to ```SERVER_URL``` and I'm guessing it's the server url for the photos ?

Reply:
Yeah, could you clarify what this server is for? If it's the s3 bucket, then it should probably have the word IMAGE or S3 in it.